Unable to publish collection to Ansible Galaxy

**Describe the bug**
A clear and concise description of what the bug is.

When attempting to publish the latest version of this project's collection, it errs out:

```
Publishing collection artifact '/home/runner/work/veeam-ansible/veeam-ansible/veeamhub-veeam-1.3.1.tar.gz' to default https://galaxy.ansible.com/api/
Collection has been published to the Galaxy server default https://galaxy.ansible.com/api/
Waiting until Galaxy import task https://galaxy.ansible.com/api/v2/collection-imports/14054/ has completed
ERROR! Galaxy import process failed: 'requires_ansible' in meta/runtime.yml is mandatory, but no meta/runtime.yml found (Code: None)
Error: Process completed with exit code 1.
```

**Expected behavior**

Ansible Collection to publish without error to Ansible Galaxy.

**Screenshots**

![image](https://user-images.githubusercontent.com/17179966/135926244-86364857-c824-4c84-954e-64babd03bc14.png)

--- a/meta/runtime.yml
+++ b/meta/runtime.yml
@@ -1,0 +1,2 @@
+---
+requires_ansible: '>=2.9'

